app/controllers/v0/sessions_controller.rb

```diff
@@ -82,7 +82,7 @@ def saml_callback
         redirect_to saml_login_redirect_url
 
         StatsD.increment(STATSD_LOGIN_NEW_USER_KEY) if @sso_service.new_login?
-        StatsD.increment(STATSD_SSO_CALLBACK_KEY, tags: ['status:success', "context:#{@sso_service.context_key}"])
+        StatsD.increment(STATSD_SSO_CALLBACK_KEY, tags: saml_callback_success_tags(@sso_service))
```

Comment:
what is "subtype" and where did this name come from? Can we call it something more descriptive?

Reply:
It's either  user_attributes.mhv_account_type,  user_attributes.dslogon_idtype, or 'idme'.  Do you have a name you'd prefer?

Reply:
hmmm, can we call subtype, "account type" and make sure to namespace the account types with an 'mhv-" or "dslogon-" prefix please?

Reply:
for dslogon, i'm not sure it is dslogon_idtype we want though is it? we want the dslogon LOA level.

Reply:
instead of "sub_type" lets call it "account_type" with the following permutations:

"mhv-basic", "mhv-advanced", "mhv-premium" (note, dont limit to these 3, instead use "mhv-" as prefix and downcase whatever MHV provides, this will be helpful to capture other randomness they might throw at us in the future.

"dslogon-1", "dslogon-2", "dslogon-3" corresponding to the dslogon_level_assurance

"idme-1", "idme-3" corresponding to the idme loa current

Reply:
for dslogon it might be interesting to also capture the dslogon_status, perhaps in a different attribute / tag though.

Reply:
Do you want the prefix when that's already part of the context tag?

Reply:
yea, maybe you can drop it -- i was just thinking that having "1" or "2" might be too little by itself. and too confusing when it is dslogon-1 vs idme-1,